src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -116,7 +115,7 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     goto(CallDone)
   }
 
-  private def registerAbortFunction(callReference: CallReference)(abortFunction: AbortFunction): Unit = {
+  private def registerAbortFunction(abortFunction: AbortFunction): Unit = {
```

Comment:
I think all the changes in this file and in CallExecutionActor.scala are already changed in my shard logging branch